Adding backup and license files for graphics + Resolve Styling comments (#37)
DynamoDS/RefineryPrimer: 01-introduction/01-01_computational-design.md

Put simply, computers are very good at **performing calculations** and **executing pre-defined steps.**

> When approaching a design computationally, the designer would focus on developing the procedure that would create a design, and not the design in itself. The process of iterating through options and data are offloaded to a machine. Saving time, money, and effort, and letting the designer focus on the creativity of the design process.

Reviewer: use `design itself` - `in` is not needed.

Reviewer: left a suggestion you can use below.

> When approaching a design computationally, the designer would focus on developing the procedure that would create a design, and not the design in itself. The process of iterating through options and data are offloaded to a machine. Saving time, money, and effort, and letting the designer focus on the creativity of the design process.

Reviewer: can't start a sentence with `Saving` and not connect it to previous one somehow.
Use `This saves time...` ... `and lets the designer focus...`

Reviewer: use suggestion below

> When approaching a design computationally, the designer would focus on developing the procedure that would create a design, and not the design in itself. The process of iterating through options and data are offloaded to a machine. Saving time, money, and effort, and letting the designer focus on the creativity of the design process.

Reviewer: ```suggestion
When approaching a design computationally, the designer would focus on developing the procedure that would create a design, and not the design itself. The process of iterating through options and data are offloaded to a machine. This saves time, money, effort, and lets the designer focus on the creativity of the design process.
```
